tokio/Cargo.toml

```diff
@@ -40,7 +41,7 @@ blocking = ["rt-core"]
 dns = ["blocking"]
 fs = ["blocking"]
 io-driver = ["mio", "lazy_static", "sync"] # TODO: get rid of sync
-io-util = ["pin-project", "memchr"]
+io-util = ["pin-project", "pin-project-lite", "memchr"]
```

Comment:
Trying out `pin-project-lite`, we probably should switch.

Reply:
(I'll work on migration to pin-project-lite, as said in discord)